### PR TITLE
Nsng 345 wip translation

### PR DESCRIPTION
**Pull request by FrantisekMichalSebestyen**

### CODE REVIEW

File: src/renderer/composables/lang.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import _ from 'lodash'
+
+const lang = ref<Lang>("sk")
+
+export const trans = (path: string) => {
+    return _.get(TRANSLATIONS[lang.value], path)
+}
+
+export function useLang() {
+    const toggle = () => { lang.value === "sk" ? lang.value = "en" : lang.value = "sk" }
```
Comment (eronisko) [inline src/renderer/composables/lang.ts:12]: I find setting more tractable than toggling. There is nothing inherently 'toggleable' about switching language (perhaps just the UI might suggest it)
```suggestion
    const setLang = (lang: 'sk' | 'en' ) => { lang.value === lang }
```

File: src/renderer/composables/lang.ts
Comment (igor-kamil) [inline src/renderer/composables/lang.ts:1]: ✨

**Review by igor-kamil — APPROVED** [review]
LGTM

File: src/renderer/views/ArtworkDetail.vue [hunk 2]
```diff
@@ -26,33 +26,40 @@
...
+          {{ trans((`gallery.${artwork.idNext}.description`)) }}
         </p>
       </article>
     </dialog>
   </div>
 </template>
 
 <script lang="ts" setup>
-import { onMounted, onBeforeUnmount, ref } from "vue";
+import { onMounted, onBeforeUnmount, ref, toRefs } from "vue";
```
Comment (eronisko) [inline src/renderer/views/ArtworkDetail.vue:45]: Unused import?

**Pull request merged**